src/components/cluster/detail/cluster_apps.js

```diff
@@ -142,32 +145,89 @@ class ClusterApps extends React.Component {
       displayApps[appMeta.category].push(appMeta);
     }
 
+    return displayApps;
+  }
+
+  imgError = e => {
+    let imageUrl = e.target.src;
+    var iconErrors = {};
+    iconErrors[imageUrl] = true;
+
+    this.setState({
+      iconErrors: Object.assign({}, this.state.iconErrors, iconErrors),
+    });
+  };
+
+  render() {
     return (
       <React.Fragment>
-        <div className='row section cluster-apps'>
-          <h3 className='table-label'>Managed Apps (Preview)</h3>
-          <p>
-            Soon you will be able to deploy managed apps like monitoring, log
-            storage, and more simply by selecting from our catalog.
-          </p>
-
-          <NavLink to={`/managed-apps/`}>
-            <Button>Browse Managed Apps</Button>
-          </NavLink>
+        <h3 className='table-label'>Installed Apps</h3>
+        <div className='row'>
+          {this.props.installedApps && this.props.installedApps.length === 0 && (
```

Comment:
Does it mean we only know if the customer can install apps if it has already app installed? 🤔

Reply:
Nope, good question, here's what it looks like when no apps are on the tenant cluster yet:

<img width="984" alt="Screenshot 2019-05-06 at 5 31 39 PM" src="https://user-images.githubusercontent.com/455309/57217309-d703e100-7024-11e9-87e8-817ea3906682.png">

Reply:
but how will it look like once we allow install apps but there is no app installed yet? If you have it under your radar and you will modify in the next PR I am fine

Reply:
Yeah I will change the text as soon as it's possible in the next PR, and will be aware of that.

Just keeping it in a 'deployable' state.